Ginger/GingerCoreNET/Run/RunsetExecutor.cs

```diff
@@ -393,11 +393,24 @@ public async Task RunRunset(bool doContinueRun = false)
 
                 //reset run       
                 if (doContinueRun == false)
-                {
+                {                    
                     if (WorkSpace.Instance.RunningInExecutionMode == false || RunSetConfig.ExecutionID == null)
                     {
                         RunSetConfig.ExecutionID = Guid.NewGuid();
                     }
+                    else
+                    {
+                        if (mSelectedExecutionLoggerConfiguration.PublishLogToCentralDB == ExecutionLoggerConfiguration.ePublishToCentralDB.Yes)
+                        {
+                            AccountReportApiHandler accountReportApiHandler = new AccountReportApiHandler(WorkSpace.Instance.Solution.LoggerConfigurations.CentralLoggerEndPointUrl);
+                            bool isExists = accountReportApiHandler.ExecutionIdValidation((Guid)RunSetConfig.ExecutionID);
+                            if(isExists)
+                            {
+                                Reporter.ToLog(eLogLevel.ERROR, string.Format("Execution id validation failed due to duplication || Runset Name : {0}, Execution Id : {1}, Please provide unique execution id.", RunSetConfig.Name , RunSetConfig.ExecutionID));
+                                return;
```

Comment:
We should not return in case duplication found, we need to show WARN on Log that duplicated ExecutionID was used and that new ExecutionID was created instead and share on Log the new ExecutionID used